kogito-bom/pom.xml

```
@@ -2128,6 +2128,11 @@
         <version>${project.version}</version>
         <classifier>javadoc</classifier>
       </dependency>
+      <dependency>
+        <groupId>org.kie.kogito</groupId>
+        <artifactId>kogito-quarkus-common-deployment</artifactId>
+        <version>${project.version}</version>
+      </dependency>
```

Please add also `sources`
```suggestion
      <dependency>
        <groupId>org.kie.kogito</groupId>
        <artifactId>kogito-quarkus-common-deployment</artifactId>
        <version>${project.version}</version>
      </dependency>
      <dependency>
        <groupId>org.kie.kogito</groupId>
        <artifactId>kogito-quarkus-common-deployment</artifactId>
        <version>${project.version}</version>
        <classifier>sources</classifier>
      </dependency>
```

---

@danielezonca Other -deployment modules don't have them, should I still add sources just for this common module?

---

Ok this is quite strange but if so fine for me to not declare it for now and maybe review this with another ticket